app_legacy/Helpers/render/game.php

```diff
@@ -69,13 +69,13 @@ function renderGameTitle(?string $title = null, bool $tags = true): string
     $matches = [];
     preg_match_all('/~([^~]+)~/', $title, $matches);
     foreach ($matches[0] as $i => $match) {
-        $category = $matches[1][$i];
+        $category = htmlspecialchars($matches[1][$i], ENT_QUOTES, 'UTF-8');
         $span = "<span class='tag'><span>$category</span></span>";
         $updateHtml($html, $match, $tags ? " $span" : '');
     }
     $matches = [];
     if (preg_match('/\s?\[Subset - (.+)\]/', $title, $matches)) {
-        $subset = $matches[1];
+        $subset = htmlspecialchars($matches[1], ENT_QUOTES, 'UTF-8');
```

Comment:
Seems like this is undesirable:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/91d90e59-3d51-4e9d-b9c9-e814dc841b70)

Reply:
Very much so, good catch. Should be resolved in https://github.com/RetroAchievements/RAWeb/pull/1604/commits/2c9f673e4500ea0d51665c0ee3617d61db751a0e.